features/guestorder.feature

```diff
@@ -0,0 +1,39 @@
+Feature: Guest order Place
+
+
+Background:Guest place Order 
+    Given The guest user will navigates to homepageurl
+    When Guest user will clicks on the respective Product
+    Then The guest user can be navigated to product details page
+
+Scenario:Guest place Order using (Negative scenario)
```

Comment:
@karthikeyan7010 From cucumber doc,
_When steps are used to describe an event, or an action. This can be a person interacting with the system, or it can be an event triggered by another system.

It’s strongly recommended you only have a single When step per Scenario. If you feel compelled to add more, it’s usually a sign that you should split the scenario up into multiple scenarios._